src/arm64/simd.h

```diff
@@ -5,11 +5,94 @@
 #include "simdprune_tables.h"
 #include "arm64/bitmanipulation.h"
 #include "arm64/intrinsics.h"
+#include <type_traits>
+
 
 namespace simdjson {
 namespace arm64 {
 namespace simd {
 
+#ifdef SIMDJSON_REGULAR_VISUAL_STUDIO
+namespace {
+// Start of private section with Visual Studio workaround
+
+
+/**
+ * make_uint8x16_t initializes a SIMD register (uint8x16_t).
+ * This is needed because, incredibly, the syntax uint8x16_t x = {1,2,3...}
+ * is not recognized under Visual Studio! This is a workaround.
+ * Using a std::initializer_list<uint8_t>  as a parameter resulted in
+ * inefficient code. With the current approach, if the parameters are
+ * compile-time constants,
+ * GNU GCC compiles it to ldr, the same as uint8x16_t x = {1,2,3...}.
```

Comment:
This is incredible. OMG.

Reply:
@jkeiser  I'd be happy to be proven wrong. Pull out Visual Studio, install the ARM thingies and try it.